File: javaScript/BreakfastTask/parlindromNumber.java

public class parlindromNumber{
public static void main(String...ekwe){

int number = 23415;

int original = number;
int reversed = 0;

for(; number > 0; number /= 10 ){
	int digit = number % 10;
	reversed = reversed * 10 + digit;
}

	if(original  == reversed){
	System.out.println(original + " number is parlindrom");
	}else{
	System.out.println(original + " not parlindrom");
	}

}
}

File: javaScript/Task/parlindromNumber.js

let number = 23415;
let original = number;
let reversed = 0;

for(; number > 0; number = Math.floor(number / 10)){
    let digit = number % 10;
    reversed = reversed * 10 + digit;
}
if(original == reversed){
    console.log(original + " number is palindrome");
    
} else{

    console.log(original + " not palindrome");
}